nilearn/plotting/img_plotting.py

```diff
@@ -963,14 +963,16 @@ def plot_glass_brain(stat_map_img,
             See http://nilearn.github.io/manipulating_visualizing/manipulating_images.html#niimg.
             The statistical map image. It needs to be in MNI space
             in order to align with the brain schematics.
-        output_file : string, or None, optional
+        output_file : string, or None, optional. 'ortho' by default.
```

Comment:
I just mention the default value already implemented. You mean I should not mention it, or that the default itself is wrong?